lock-api/src/test/java/com/palantir/lock/client/TransactionStarterTest.java

```diff
@@ -78,19 +96,19 @@ public void after() {
 
     @Test
     public void shouldDeriveStartTransactionResponseFromBatchedResponse_singleTransaction() {
-        StartTransactionResponseV4 startTransactionResponse = getStartTransactionResponse(12, 1);
+        StartTransactionResponseV5 startTransactionResponse = getStartTransactionResponse(12, 1);
```

Comment:
as a meta point, if `StartTransaction(Request|Response)` is not transferred over the wire, can we get rid of the V5 etc?

Reply:
Yup, will open a follow-up PR